le.go

```diff
@@ -138,9 +138,21 @@ func (logger *Logger) Flags() int {
 
 // Output does the actual writing to the TCP connection
 func (logger *Logger) Output(calldepth int, s string) error {
-	_, err := logger.Write([]byte(s))
-
-	return err
+	var err error
+	for {
```

Comment:
This has the potential to be an infinite loop if the connection is dead and unable to recover (eg. server is down).
Retry once or with an exponential backoff maybe?